lib/finders/people.json

```diff
@@ -0,0 +1,30 @@
+{
+  "base_path": "/government/people",
+  "title": "All ministers and senior officials on GOV.UK",
+  "description": "",
+  "locale": "en",
+  "document_type": "finder",
+  "schema_name": "finder",
+  "publishing_app": "whitehall",
+  "rendering_app": "finder-frontend",
+  "details": {
+    "document_noun": "person",
+    "default_order": "title",
+    "facets": [],
+    "filter": {
+      "format": "person"
```

Comment:
`format` has been deprecated in favour of `schema_name` and `document_type`. Is this still the correct facet to filter on?